webapp/components/admin_console/storage_settings.jsx

```diff
@@ -87,7 +87,7 @@ export default class StorageSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.image.storeTitle'
-                            defaultMessage='Store Files In:'
+                            defaultMessage='File Storage System:'
```

Comment:
Let's discuss, 

1. "File Storage System" sounds close, but there's something about that phrasing that feels weird to me
2. We should figure out how to add help text here. @esethna could you take a shot at the change to add help text here? You can probably grok the pattern, or maybe ask a dev while at the offsite? 

![image](https://cloud.githubusercontent.com/assets/177788/16301728/f5a04620-3914-11e6-8fba-fedd0e6044cf.png)